docarray/array/storage/sqlite/seqlike.py

```diff
@@ -80,3 +81,15 @@ def __eq__(self, other):
             and type(self._config) is type(other._config)
             and self._config == other._config
         )
+
+    def extend(self, docs: Iterable['Document']) -> None:
+
+        self_len = len(self)
+        for doc in docs:
```

Comment:
use `cursor.executemany()` in the sqlite get better performance.

Reply:
I added it and removed it since I did not see any benefits